packages/components/Callout/macos/CalloutView.swift

```diff
@@ -26,6 +26,8 @@ class CalloutView: RCTView, CalloutWindowLifeCycleDelegate {
 		}
 	}
 
+    @objc public var setInitialFocus: Bool = false
```

Comment:
Hmm would setting the default value to true preserve the existing behaviour if setInitialFocus isn't set?

Reply:
It would, but I prefer the default to be false (AKA, as if the prop was never passed in). Both Menu and ContextualMenu set the prop to true so I don't think we'll see a regression there.